runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessAttendedVariables.ts

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from '#src-app/store';
+import { currentProcessSelector } from '#src-app/store/slices/Process';
+
+const useProcessAttendedVariables = () => {
+    const { isAttended } = useSelector(currentProcessSelector);
+    const { passedInVariables } = useSelector((state) => state.process.modeler);
+
+    const attendedVariables = isAttended
+        ? [{name: passedInVariables[0]}, {name: passedInVariables[1]}]
```

Comment:
why  ? [{name: passedInVariables[0]}, {name: passedInVariables[1]}], what if we have 3 or more attended vars